eclair-core/src/main/scala/fr/acinq/eclair/channel/Helpers.scala

```diff
@@ -423,21 +424,26 @@ object Helpers {
       }
     }
 
-    def firstClosingFee(commitments: Commitments, localScriptPubkey: ByteVector, remoteScriptPubkey: ByteVector)(implicit log: LoggingAdapter): Satoshi = {
+    def firstClosingFee(feeratePerKw: Long, commitments: Commitments, localScriptPubkey: ByteVector, remoteScriptPubkey: ByteVector)(implicit log: LoggingAdapter): Satoshi = {
```

Comment:
`feeratePerKw` should be the last parameter here imho

Reply:
Agreed, reorg'd in f0649dd